### ISSUE TITLE
BT Pay is not working using safetynet-fix

### ISSUE THREAD

**Issue by ssesdd** [kdrag0n/safetynet-fix#275]
BT Pay, a bank payment app is not working on a Samsung S20+ with last version Magisk 25.2 and safetynet-fix enabled, Zygisk is enabled with the app in denylist, magiskhide props is enabled
Google Play services is removed.

BT Pay available here
https://apkcombo.com/bt-pay/ro.btrl.pay/

**Comment by BLADR-ONE** [comment]
Same here dude...
You just need to pass safetynet but it seems i cannot do so.
I've managed to do it in the past with magisk hide props config but rn it isn't working anymore...
I think the problem is play servicies unchecking automaicly from denylist on reboot.
I have a nubia red magic 5g
Please anybody that knows why help me...

**Comment by ssesdd** [comment]
Yes, correct, SafetyNet Test says: fail (Google Play services not available). I've removed them but, if I understand correctly, Safetynet-fix should take care of that in version 2.4.0, right? https://github.com/kdrag0n/safetynet-fix/releases 
Anyway, Revolut is working, but only BT Play not

**Comment by BLADR-ONE** [comment]
I've managed to fix it and pass safetynet by pairing safetynet fix by kdrag0n with magisk hide props config.
You must pick a certified fingerprint from magisk hide props config (you can do that in terminal or termux) but don't pick another android version bc it will brick your device.
Then you install USF by kdrag0n and reboot.
Lastly be sure that SElinux is set on enforcing, and be sure that id dosen't change.(use sestatus and setenforce 1)
Then it should pass saetynet.
BT pay unfortunately blacklists your device so you need device ID changer app to change your dev ID for BT pay than clear data and frce stop and restart the device. now it should work

credits for helping me figure it out @IRules

**Comment by ssesdd** [comment]
> I've managed to fix it and pass safetynet by pairing safetynet fix by kdrag0n with magisk hide props config. You must pick a certified fingerprint from magisk hide props config (you can do that in terminal or termux) but don't pick another android version bc it will brick your device. Then you install USF by kdrag0n and reboot. Lastly be sure that SElinux is set on enforcing, and be sure that id dosen't change.(use sestatus and setenforce 1) Then it should pass saetynet. BT pay unfortunately blacklists your device so you need device ID changer app to change your dev ID for BT pay than clear data and frce stop and restart the device. now it should work
> 
> credits for helping me figure it out @IRules

You did this with Google services, G. play, G. framework removed or in place?

**Comment by BLADR-ONE** [comment]
with them installed

**Comment by ssesdd** [comment]
So most probably that's the difference. I don't want any google installed, all are removed. I also prefer not to install microG just for BT Pay (and it's not clear for me if that will help, anyway) because everything else needed is working well

**Comment by AshKxK** [comment]
> I've managed to fix it and pass safetynet by pairing safetynet fix by kdrag0n with magisk hide props config. You must pick a certified fingerprint from magisk hide props config (you can do that in terminal or termux) but don't pick another android version bc it will brick your device. Then you install USF by kdrag0n and reboot. Lastly be sure that SElinux is set on enforcing, and be sure that id dosen't change.(use sestatus and setenforce 1) Then it should pass saetynet. BT pay unfortunately blacklists your device so you need device ID changer app to change your dev ID for BT pay than clear data and frce stop and restart the device. now it should work
> 
> credits for helping me figure it out @IRules

I am using Mi 10T Lite with Superior OS(masked by default with Pixel 2). It runs android 13, last android versions available on Magisk Prop Config are for A12. tried to use Pixel 3 Xl A12, and it fails even safetynet now. reversed and tried BT Pay and said that my phone has unlocked bootloader. What fingerprint did you use to bypass?
